tests/TestClassDataDetailView.php

```diff
@@ -69,6 +69,38 @@ class TestClassDataDetailView
 		'mobile',
 	];
 
+	/** @var DataViewSimilarEstates */
+	private $_pDataViewSimilarEstates = null;
+
+	/**
+	 *
+	 */
+
+	public function test__wakeup()
+	{
+		$pInstance = new DataDetailView();
+
+		// old versions didn't have $_pDataViewSimilarEstates, but it must always be set
+		$pClosure = function() {
```

Comment:
A way to solve this without `Closure::bind()` would be much nicer, for example by using a fixed string of that serialized object.